data/Text.csv

```diff
@@ -116,5 +116,5 @@ T00075,The Uses of Literature,Life in the Socialist Chinese Literary System,en,P
 T00075,,,zh,P0150,G19,PL0072,2000,Princeton Univ. Press,,,NULL,NULL,,2017-09-22,MG,2017-09-22,MG
 T00076,L‘Étranger,,fr,P0151,G04,PL0003,1961,Shanghai renmin chubanshe,,yes,T00005,580,According to Wang Shiqiang's research this piece was one of the most influential yellow-covered books for the generation of pre-''misty'' poets. Translated by Meng An 孟安. According to Wang Shiqiang the Novel was published by Shanghai wenyi chubanshe.,2017-09-22,MG,2017-09-22,MG
 T00076,局外人,,zh,P0151,G04,PL0003,1961,Shanghai renmin chubanshe,,yes,T00005,580,According to Wang Shiqiang's research this piece was one of the most influential yellow-covered books for the generation of pre-''misty'' poets. Translated by Meng An 孟安. According to Wang Shiqiang the Novel was published by Shanghai wenyi chubanshe.,2017-09-22,MG,2017-09-22,MG
-T00077,Dream of the Red Chamber,,en,P0152,G04,PL0001,1962,CHIA-YI,,,T00044,372,,2017-09-22,MG,2017-09-22,MG
-T00077,红楼梦,,zh,P0152,G04,PL0001,1962,,,,T00044,372,,2017-09-22,MG,2017-09-22,MG
+T00077,Dream of the Red Chamber,,en,P0152,G04,PL0001,1962,Cannot find any Hongloumeng published in 1962; but in 1957 and 1982 was published by 人民文学出版社,,,T00044,372,,2017-09-22,MG,2017-09-24,CW
```

Comment:
Please keep the data columns clean. The best place for these kind of notes is an issue on GitHub, alternatively you can put these kind of annotations into the notes field.